credoai/integration.py

```diff
@@ -71,21 +67,21 @@ class Metric(Record):
     def __init__(self,
             metric_type,
             value,
-            dataset_label,
+            dataset_variant = None,
             **metadata):
         super().__init__('metrics', **metadata)
         self.metric_type = metric_type
         self.value = value
-        self.dataset_label = dataset_label
+        self.dataset_variant = dataset_variant
         self.config_hash = dict_hash({k:v for k,v in self.__dict__.items() 
                                       if k!='value'})
```

Comment:
This hash value will be always different because of creation_time. I think it should be restricted to the several attributes like metric_type, name, and dataset_variant_id.

Also name is highly recommended to be included. The web app's metric view will be confusing without the name.

Reply:
Nice catch - I'll remove `creation time`. And I'll have a name always supplied.